crypto/rand/rand_lib.c

```diff
@@ -566,6 +570,30 @@ size_t rand_pool_bytes_remaining(RAND_POOL *pool)
     return pool->max_len - pool->len;
 }
 
+static int rand_pool_resize(RAND_POOL *pool, size_t len)
```

Comment:
Since `len` is incremental and not the final size, the name `rand_pool_resize()` is misleading.
A name like `rand_pool_grow()` or  `rand_pool_enlarge()` would be more appropriate.

Reply:
[embiggen](https://www.lexico.com/en/definition/embiggen)? :)

Reply:
What about `rand_pool_check_space()` since it doesn't always enlarge the storage?

Reply:
How come it doesn’t surprise me that you are a  Simpsons fan? ;-)

Reply:
It does grow ‚len‘.

Reply:
I meant pool->len. (Sorry I‘m on my mobile.)

Reply:
The surprise is that I'm not much of a fan...

Reply:
I've switched to `_grow`